src/stdio_mgr/stdio_mgr.py

```diff
@@ -130,10 +132,22 @@ def append(self, text):
         """
         pos = self.tell()
         self.seek(0, self.SEEK_END)
```

Comment:
Huh, it's not necessary to do the seek on `self._buf` or `self.buffer` here?  I guess `TextIOWrapper` passes the seek through to the correct wrapped object?